ipn/backend.go

```diff
@@ -21,31 +21,21 @@ import (
 
 type State int
 
+//go:generate go run github.com/nikolaydubina/go-enum-encoding@v1.6.0 -type=State -string
```

Comment:
conventionally these are not attached to an AST node that'd generate public godoc.

Reply:
I tested, these do not appear in godoc. I believe `//go:generate` directive does not qualify as a comment as is skipped by godoc.

do you see any problem? would you still insist having empty lines between var/const block and `//go:generate`?

<img width="608" alt="image" src="https://github.com/user-attachments/assets/487665ef-1744-421c-a68e-5d6d0e571338">
<img width="457" alt="image" src="https://github.com/user-attachments/assets/3093dd1e-f27d-4b7f-bc5a-4290bfd9727e">